src/gas/gas-util.ts

```diff
@@ -17,9 +17,7 @@ const makeClientIdHeader = (clientId: string) => ({ 'X-Client-Id': clientId });
  * @returns The decimal string GWEI amount.
  */
 export function normalizeGWEIDecimalNumbers(n: string | number) {
-  const numberAsWEIHex = gweiDecToWEIBN(n).toString(16);
-  const numberAsGWEI = weiHexToGweiDec(numberAsWEIHex).toString(10);
-  return numberAsGWEI;
+  return weiHexToGweiDec(gweiDecToWEIBN(n).toString(16));
```

Comment:
Simplified because 1) `weiHexToGweiDec` already returns a string, so there is no need to call `toString(10)` on it and 2) we can do this in one line.